schemas/common/essa.shema.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
`essa.shema.json` should be `essa.schema.json`

Reply:
good catch